### PR TITLE
Changes to Alien Artifacts and Alien Containers

### PR DESCRIPTION
**Pull request by ubertpendragon**
Changes the alien Artifacts. 

Made artifacts more "dangerous". Previously artifacts didnt have much of an effect, being easily disposed of in the ballast and forgotten. These changes try to combat that. 

Changes to artifacts
- Faraday artifact EMP area increased by 4x, EMP value decreased by 20x, added a discharge coil effect out of water.
- Thermal artifact now emits steam in water that burns and damages walls, density reduced. 
- Nasonov artifact now also disrupts the sonar. Added particle effect.
- Psychosis artifact now generates watcher's gaze periodically, causing psychosis/nausea and strengthening nearby monsters
- Sky artifact now drains water on top of draining oxygen, which will mess with your ballasts. **HIGHLY EXPERIMENTAL, make sure it doesnt drain water from ruins**
- Doubled the price of all artifacts. At a base price of 1000, bringing back an artifact would only net 600mk at most. Moloch bones are less rare and worth just as much.

Changes to containers:
- Added the <artifactcontainer> tag to small, medium and large alien crates to disable artifact effects. Artifacts would sometimes negatively affect enemies inside ruins (fractal guardians would get shocked by the EMP).
- Minor changes to the incubation bubble, needs to be "popped" by hitting it, uses the broken sprite, added particles

https://user-images.githubusercontent.com/73229309/224340700-934290da-de7b-4106-8000-a5b2abf11510.mp4

https://user-images.githubusercontent.com/73229309/224340199-b1869ad5-a4fd-48ca-96d4-0dd85c7f6921.mp4

### CODE REVIEW

**Comment by ubertpendragon** [comment]
Un-broke the sky artifact, hopefully this is the one

File: alienitems.xml [hunk 1]
```diff
@@ -0,0 +1,2001 @@
...
+            <Conditional hastag="neq artifactcontainer" targetcontainer="true" />
+            <Conditional Voltage="0.0" targetcontainer="true" />
+            <Explosion range="800.0" empstrength="0.25" camerashake="2" stun="0" force="2.0" flames="false" smoke="false" shockwave="false" sparks="true" underwaterbubble="false" />
+            <!-- separate smaller explosion for EMP affliction so it doesn't fry guardians -->
+            <Explosion range="200.0" camerashake="0" stun="0" force="0" flames="false" smoke="false" shockwave="false" sparks="false" underwaterbubble="false" >
+              <Affliction identifier="emp" strength="25" multiplybymaxvitality="true" />
+            </Explosion>
+            <sound file="Content/Items/Alien/AlienCoil1.ogg" range="1000" />
+            <ParticleEmitter particle="faradayfx" anglemax="360" distancemax="80" scalemultiplier="2.6,2.6" particleamount="40" />
+      </StatusEffect>
```
Comment (Regalis11) [inline alienitems.xml:131]: Broken indentation

File: alienitems.xml [hunk 2]
```diff
@@ -0,0 +1,2001 @@
...
+        <Fire />
+      </StatusEffect>
+      <StatusEffect type="Always" target="This">
+        <Conditional hastag="neq artifactcontainer" targetcontainer="true" />
+        <Conditional voltage="lte 0.0" targetcontainer="true" />
+        <ParticleEmitter particle="thermalfx" anglemax="360" distancemax="5" particlespersecond="40" />
+        <sound file="Content/Items/Alien/AlienArtifactLoop1.ogg" range="500.0" loop="true" />
+      </StatusEffect>
+    </Holdable>
+        <!-- Emits steam periodically when in water, burns. Had to use Lightcomponent to make it work -->
```
Comment (Regalis11) [inline alienitems.xml:82]: Broken indentation

File: alienitems.xml [hunk 3]
```diff
@@ -0,0 +1,2001 @@
...
+            <ParticleEmitter particle="nasonovsparkles" anglemax="360" distancemax="10" velocitymin="10" velocitymax="60" emitinterval="0.55" particleamount="5" particlespersecond="5" />
+            <sound file="Content/Items/Alien/AlienArtifactLoop1.ogg" range="500.0" loop="true" volume="0.5"/>
+        </StatusEffect>
+        <StatusEffect type="Always" target="This" delay="1.1" stackable="false" SoundRange="50000" setvalue="true" >
+            <Conditional hastag="neq artifactcontainer" targetcontainer="true" />
+            <Conditional Voltage="0.0" targetcontainer="true" />
+            <ParticleEmitter particle="nasonovfx" anglemax="360" particleamount="3" />
+            <sound file="Content/Items/Alien/Alien_ArtifactHolderLoop.ogg" range="1000.0" loop="true" volume="3"/>
+        </StatusEffect>
+    </Holdable>
```
Comment (Regalis11) [inline alienitems.xml:189]: Broken indentation

File: alienitems.xml [hunk 4]
```diff
@@ -0,0 +1,2001 @@
...
+      <StatusEffect type="OnActive" target="This" IsOn="False" delay="5" stackable="False" />
+      <StatusEffect type="OnActive" target="This" checkconditionalalways="true" >
+        <Conditional hastag="neq artifactcontainer" targetcontainer="true" />
+        <Conditional Voltage="lte 0.0" targetcontainer="true" />
+        <ParticleEmitter particle="bubbles" particlespersecond="5" anglemin="60" anglemax="120" scalemin="1" scalemax="2" velocitymin="5" velocitymax="100" copyentityangle="false" drawontop="true"/>
+        <ParticleEmitter particle="steamspray" particlespersecond="30" anglemin="60" anglemax="120" scalemin="1" scalemax="2" velocitymin="300" velocitymax="500" copyentityangle="false" highqualitycollisiondetection="true" drawontop="true" />
+        <sound file="Content/Items/Weapons/FlameThrowerLoop.ogg" type="OnUse" range="700.0" loop="true" />
+        <!-- burns via explosion -->
+        <Explosion range="200" structuredamage="0.1" itemdamage="0" smoke="False" flames="false" force="0" sparks="false" shockwave="False" underwaterbubble="false" camerashake="2" flash="false" playtinnitus="false" >
```
Comment (Regalis11) [inline alienitems.xml:94]: It seems like executing this every frame might be bad for performance. Could probably be run once per second or so with no noticeable effect on the behavior.

**Review by Regalis11 — CHANGES_REQUESTED** [review]

File: alienitems.xml (hunk 4 above)
Comment (ubertpendragon) [reply alienitems.xml:94]: Unfortunately adding a 1 second interval breaks the sound and the particles, a 0.1 second interval keeps it "smooth". With a bigger interval and you see gaps between particles and it sounds like a steam engine

I have tried turning it into a repair tool that uses itself to no avail, it doesnt damage the holder

Comment (ubertpendragon) [reply alienitems.xml]: Split the Statuseffect into two, assuming the explosion was the most performanceh hungry. Not sure if it's better.

**Comment by Regalis11** [comment]
Thank you for the contribution! :) I moved this to a [PR in our private repo now](https://github.com/Regalis11/Barotrauma-development/pull/4358)

**Pull request closed without merging**